TicketBookingBackend/TicketBookingAPI/TicketBooking.Services/Classes/AuthenticationService.cs

```diff
@@ -0,0 +1,115 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using TicketBooking.Domain;
+using TicketBooking.Models;
+using TicketBooking.Repository.Interfaces;
+using TicketBooking.Services.Interfaces;
+using TicketBookingDomain;
+
+namespace TicketBooking.Services.Classes
+{
+    public class AuthenticationServices : IAuthenticationServices
```

Comment:
AuthenticationServices -- AuthenticationService

Reply:
initially i kept it there on purpose, because there was one IAuthenticationService in aspnetcore.Authentication, but don't need to keep it that way, resolved.